#.gitignore#

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
Is this file needed?

Reply:
Not even sure how #.gitignore# appeared, will delete